opal/models.py

```diff
@@ -1684,6 +1707,19 @@ def explicit_access_only(self):
         return any(r for r in all_roles if r == "scientist")
 
 
+class Stage(TrackedModel):
+    """ An episode can have many stages, but should have no more than
+        one with no stop date any time.
+
+        This is expected to be set by the episode_category
+        which determines what stages are available to an episode.
+    """
+    episode = models.ForeignKey(Episode, null=False)
+    started = models.DateTimeField()
+    stopped = models.DateTimeField(blank=True, null=True)
+    value = models.CharField(max_length=256)
+
```

Comment:
Is it possible to have e.g. episode and stopped unique_together? 
Will this prevent two stages for a single episode without an end date?